OpenSim/Common/Test/testComponentInterface.cpp

```diff
@@ -690,6 +690,11 @@ void testMisc() {
 
     cout << "Adding world3 to theWorld" << endl;
     theWorld.add(world3.clone());
+    // We now use absolute paths for the connectee name, and adding world3 to
+    // theWorld makes the previously-stored absolute path incorrect.
+    // TODO add() should update absolute connectee paths; no longer necessary.
```

Comment:
What does the "no longer necessary" refer to here?

Reply:
Ah thanks for the catch. I can remove these changes now that connectee paths are updated in `add...()`.